### ISSUE TITLE
Firefox Home Icons are Too Thin, Small and Lack Contrast

### ISSUE THREAD

**Issue by aakash-kap** [mozilla-mobile/firefox-ios#8996]
### User Problem
> Icons for sections like Bookmarks, History, Downloads, etc. on Firefox Home are a bit too small and thin, making them a bit unnoticeable, especially if the user wears spectacles. In addition, they are not contrasted well while in Dark Mode (especially the History Icon). Benefit : Better Accessibility and a more cohesive UI

### Ideas for Solution
> Replace the icon assets with thicker, higher contrast and bigger size icons for all icons on Firefox Home

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-3080)

**Comment by lmarceau** [comment]
Closing since the library section doesn't exists anymore on home page
